Swag/Paths/Video/Video.yaml

```diff
@@ -0,0 +1,36 @@
+# GET /video
+AllVideos:
+  tags:
+    - Video
+  summary: Returns list of Videos.
+  operationId: getAllVideo
+  description: This end point returns list of Videos that CodingGarden Community has. # Suggestions?
```

Comment:
This endpoint will ultimately be a feed of The Coding Garden YouTube channel vids.

Reply:
Updated.

Reply:
I wouldn't word it that way I was just letting you know what it will be. I would use something like "This endpoint returns a list of videos from The Coding Garden YouTube channel."

Reply:
Oh. Sorry. is this specific to "Coding Garden with CJ" channel? If that's that case, I 'll mention that? Or are there other channels too?

Reply:
Updated. Sorry about the last one. IDK what I was thinking.